docker/README.md

```diff
@@ -1,18 +1,18 @@
+The Dockerfile is used to build and package Intel data center GPU driver container images for OCP releases. Since OCP follows a regular [release cycle](https://docs.openshift.com/dedicated/osd_architecture/osd_policy/osd-life-cycle.html) and releases new versions frequently, the RHCOS kernel used by each release may be different. This project must regularly release the driver container image to match the new kernel version used by each OCP release. We refer to this scenario as the **kernel upgrade scenario**.
 
-In the Dockerfile [Red Hat Driver Toolkit (DTK)](https://github.com/openshift/driver-toolkit) is used as the base image to build the kernel driver modules. DTK integrates all the necessary packages and tools for building kernel modules in the container environment on OCP cluster. Moreover, the proper version of the DTK image is shipped with the OCP release which verifies the kernel modules built match the RHCOS kernel shipped with the OCP version. 
+In the Dockerfile, [Red Hat Driver Toolkit (DTK)](https://github.com/openshift/driver-toolkit) is used as the base image to build the kernel driver modules. DTK integrates all the necessary packages and tools for building kernel modules in the container environment on OCP cluster. Moreover, the proper version of the DTK image is shipped with the OCP release which verifies the kernel modules built match the RHCOS kernel shipped with the OCP version. 
 
-`Note: ARG DTK_AUTO in the Dockerfile will be replaced automatically with the DTK image by KMM operator.`
+`Note: ARG DTK_AUTO in the Dockerfile will be replaced automatically with the appropriate DTK image by KMM operator.`
 
-Intel GPU Drivers uses Dynamic Kernel Module Support (DKMS) to release the RPM packages and on-premises build the drivers modules on the RHEL server. With this Dockerfile, [Linux Kernel Build System](https://docs.kernel.org/kbuild/index.html) is used to build the driver modules.
+Intel GPU Drivers uses Dynamic Kernel Module Support (DKMS) to release the RPM packages and on-premise build the drivers modules on the RHEL server. With this Dockerfile, [Linux Kernel Build System](https://docs.kernel.org/kbuild/index.html) is used to build the driver modules.
 
-Multiple drivers are compiled and packaged into the single driver container image. To make sure the right drivers release combination are packaged, Intel GPU Driver version is used to select the version of each driver.
+Multiple drivers are compiled and packaged into the single driver container image. To ensure the correct driver release combination is packaged, Intel GPU Driver version is used to select the version of each driver.
```

Comment:
Since we have several drivers packaged into the image. 
Should we use the correct "drivers" release?

Reply:
The "driver release combination" implies that a combination consists of several driver releases.

Reply:
@uMartinXu "driver" is correct usage here.

Reply:
Thanks! :=)